.github/workflows/multi-gpu-ci.yml

```diff
@@ -0,0 +1,30 @@
+name: horovod (2GPU)
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+    tags:
+      - v*
+  pull_request:
+    branches: [ main ]
+    types: [opened, synchronize, reopened, closed]
+
+jobs:
+  gpu-ci:
+    runs-on: 2GPU
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Run tests
+      run: |
+        ref_type=${{ github.ref_type }}
+        branch=main
+        if [[ $ref_type == "tag"* ]]
+        then
+          git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +refs/heads/release*:refs/remotes/origin/release*
```

Comment:
Just curious, asking for my own edification: what's different in `protocol.version=2`?

Reply:
this was copied from other places that have this pattern. now available in [the branch-name action here](https://github.com/NVIDIA-Merlin/.github/tree/main/actions/branch-name). The reason for the `protocol.version=2` was to match what [the GitHub actions/checkout](https://github.com/actions/checkout/blob/v3.3.0/src/git-command-manager.ts#L205) does in the fetch. It may no longer be strictly necessary since the default protocol is v2 with recent versions of git.

Reply:
It looks like v2 enables more commands to be sent over a single connection, among other things: https://git-scm.com/docs/protocol-v2